Windows/GEDebugger/CtrlDisplayListView.cpp

```diff
@@ -306,6 +310,10 @@ void CtrlDisplayListView::onMouseDown(WPARAM wParam, LPARAM lParam, int button)
 
 void CtrlDisplayListView::onMouseUp(WPARAM wParam, LPARAM lParam, int button)
 {
+	if (!validDisplayList) {
```

Comment:
nit: Inconsistent bracing style, relative to nearby code

Reply:
it's a mix of bracing styles in this file, and we're moving towards this one. I'm keeping it.

Reply:
Makes sense!